grow/testing/testing.py

```diff
@@ -7,16 +7,17 @@
 TESTDATA_DIR = os.path.abspath(os.path.join(_here, 'testdata'))
 
 
-def create_test_pod_dir():
+# pod_dir is relative to testdata/ dir
+def create_test_pod_dir(pod_dir='pod'):
```

Comment:
The term we use for what you're calling `pod_dir` is `root`. (See an example here: https://github.com/grow/grow/blob/master/grow/commands/run.py#L33). Just for consistency can you switch `pod_dir` to `root` here?